src/libs/actions/OnyxUpdateManager.ts

```diff
@@ -41,9 +41,14 @@ export default () => {
     Onyx.connect({
         key: ONYXKEYS.ONYX_UPDATES_FROM_SERVER,
         callback: (value) => {
-            // When the OpenApp command hasn't finished yet, we should not process any updates.
+            // When value is set to null (which we do once we finish processing this method), we can just return early.
+            // Alternatively, if isLoadingApp is positive, that means that OpenApp command hasn't finished yet, so we
+            // should not process any updates that are being set.
```

Comment:
This still doesn't quite make sense. I'll ask a couple of questions maybe and then that will help you craft a comment that answers the questions.

- Why should updates not be processed while OpenApp is pending? (Answer, I think: The base state of the app should be created before the first onyx updates are applied to it, or else there is a race condition where the first onyx updates get applied before the app is in its initial state and the first onyx updates are overwritten.)

Reply:
Agree, useful comments say why we do things, saying what we do is kind of implied by the code (but it's fine to explain it in the comment too)